object_database/web/cells/cells.py

```diff
@@ -1811,19 +1891,20 @@ def getChild(self):
 
     def recalculate(self):
         with self.view():
-            self.children = {"____child__": self.getChild()}
+            childCell = self.getChild()
+            self.children = {"____child__": childCell}
+            self.namedChildren['child'] = childCell
 
 
 class Subscribed(Cell):
     def __init__(self, f):
         super().__init__()
 
-        self.f = f
+        self.f = f  # What is f? A lambda?
```

Comment:
yes, it's a lambda, and we should rename it. IT's the function that generates the actual cell.